### PR TITLE
feat: humans.txt (and disallow AI crawler bots)

### PR DESCRIPTION
**Pull request by andreasgrafen**
This PR adds a humans.txt file according to: [Humans TXT: We Are People, Not Machines.](https://humanstxt.org/).

For now I've just mirrored the structure of the [community](https://catppuccin.com/community) page but the format allows us to extend the information. For me I have added my personal website and my Discord handle for example.

### CODE REVIEW

File: public/humans.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/* USERSTYLES STAFF */
+Bell
+GitHub: https://github.com/isabelroses
+Website: https://isabelroses.com
+From: UK
+
+uncenter
+GitHub: https://github.com/uncenter
+From: USA
```
Comment (uncenter) [inline public/humans.txt:51]:
```suggestion
uncenter
GitHub: https://github.com/uncenter
Website: https://uncenter.dev
From: USA
```

File: public/humans.txt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+GitHub: https://github.com/taka0o
+From: Canada
+
+Isabel
+GitHub: https://github.com/isabelincorp
+From: USA
+
+Pigeon:
+GitHub: https://github.com/backwardspy
+From: UK
```
Comment (backwardspy) [inline public/humans.txt:37]:
```suggestion
pigeon
GitHub: https://github.com/backwardspy
Website: https://pigeon.life
From: UK
```

File: src/layouts/Home.astro [hunk 3]
```diff
@@ -39,6 +39,7 @@ const isVercelProd = import.meta.env.VERCEL_PROD;
 
     <link rel="icon" type="image/svg+xml" href="/favicon.png" />
 
+    <link type="text/plain" rel="author" href="http://domain/humans.txt" />
```
Comment (isabelroses) [inline src/layouts/Home.astro:42]: Should `href="http://domain/humans.txt"` not be `/humans.txt`?

File: public/humans.txt [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+From: USA
+
+/* MODERATOR */
+Nyx
+GitHub: https://github.com/nyxkrage
+E-Mail: carsten@kragelund.me
+From: Denmark
+
+Etzelia
+GitHub: https://github.com/jolheiser
```
Comment (jolheiser) [inline public/humans.txt:62]:
```suggestion
GitHub: https://github.com/jolheiser
Website: https://jolheiser.com
```

File: src/layouts/Home.astro (hunk 3 above)
Comment (andreasgrafen) [reply src/layouts/Home.astro:42]: oops.. That's also for the other layout too. I totally missed that. D:

**Pull request merged**